submissions/Oleksandr-Heleta/a-tiny-js-world/index.js

```diff
@@ -0,0 +1,83 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+const dog = {
+   species: 'dog',
+   name: 'Bob',
+   gender: "male",
+   legs: 4,
+   hands: 0,
+   say: "bark"
+};
+
+const cat = {
+   species: 'cat',
+   name: 'Kitty',
+   gender: "female",
+   legs: 4,
+   hands: 0,
+   say: "meow"
+};
+
+const man = {
+   species: 'human',
+   name: 'John',
+   gender: "male",
+   legs: 2,
+   hands: 2,
+   say: "Hello Jenny!"
+};
+
+const woman = {
+   species: 'human',
+   name: 'Jenny',
+   gender: "female",
+   legs: 2,
+   hands: 2,
+   say: "Hi John!"
+};
+
+const catWoman = Object.create(cat);
+catWoman.species = "cat-woman";
+catWoman.name = "Jini";
+catWoman.legs = 2;
+catWoman.hands = 2;
+
+const persons = [dog, cat, man, woman, catWoman];
+const properties = ['species', 'name', 'gender', 'legs', 'hands', 'say']
+
+function createString(persons, properties) {
```

Comment:
Avoid generic words and data types when naming variables and functions. Try using domain-specific terms.
What would this be in the real world?